_posts/2024-09-17-pyopensci-sustainability-plan-2024.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: single
+title: "A Blueprint for the Future: pyOpenSci's Sustainability Model"
+excerpt: "pyOpenSci is exploring models to become more sustainable which means bringing in funding beyond just grants. Learn more about what we are exploring here."
```

Comment:
I would reframe a bit to reiterate how valuable our grant founders are.

```suggestion
excerpt: "pyOpenSci is exploring various sustainability methods to supplement our valued grant funding. Learn more about what we are exploring."
```

Reply:
good suggestion!! thank you! I want to be really careful with the language used here!!